gen/calc.py

```diff
@@ -387,7 +387,7 @@ def calculate_config_yaml(user_arguments):
 def calculate_mesos_isolation(enable_gpu_isolation):
     isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,'
                  'docker/runtime,docker/volume,volume/sandbox_path,posix/rlimits,'
-                 'com_mesosphere_MetricsIsolatorModule')
+                 'namespaces/pid,com_mesosphere_MetricsIsolatorModule')
```

Comment:
Sometimes the order matters (but it doesn't for this one). These are already not in alphabetical order (look at`volume/sandbox_path` and networks/cni`, and `filesystem/linux` for example). I know that `filesystem/linux` has to come before the docker ones (and the gpu one which gets added later based on the config). I'm fine with moving it if you want, just not sure where it best fits.